sympy/functions/elementary/hyperbolic.py

```diff
@@ -73,7 +73,14 @@ class sinh(HyperbolicFunction):
     r"""
     The hyperbolic sine function, `\frac{e^x - e^{-x}}{2}`.
```

Comment:
As you're going through, you should fix instances of single backticks to either be dollars for math (like in this case), or double backticks for code.

Reply:
Yes. Will keep this in mind.